packages/ui-components/src/YayNay.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+import { Bar, BarChart, CartesianGrid, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
+
+interface IProps {
```

Comment:
The ideal way would be to actually rename this to `YayNayProps` and export it (like SUI does), so that other components can use this interface as well. But just a small note for the future, no action needed now.